data/wanderer/wanderer outfits.txt

```diff
@@ -624,18 +624,18 @@ outfit "Wanderer Outfits License"
 	series "Licenses"
 	index 09010
 	thumbnail "outfit/wanderer outfits license"
-	description "Spoke with a Wanderer ambassador named Iktat Rek using the translation device and received permission to purchase some of their technology, but not their ships."
+	description "This oddly shaped dish holds a perfectly preserved set of seeds of one Poaognod Nidadolen, a hardy species of grass that the Wanderers use during their initial terraforming efforts to gauge how well a world is responding to their methods."
```

Comment:
I'm not sure about this one.
The Wanderer Campaign mentions the Wanderers working with local vegetation in their early efforts on the Korath worlds, rather than using species they're bringing with them.

Reply:
Perhaps something like this instead?

`description "This oddly shaped dish holds a collection of perfectly preserved seeds, each set hailing from one of the many regions of space the Wanderers have restored. The most recent addition is that of the Poaognod Nidadolen, a hardy species of grass used to gauge how well a world is responding to their methods."`

reticent-rem has voiced in the Discord that some other naming scheme for the plant names would be better, so the current ones could just be replaced.

Reply:
@Quantumshark what do you think?

Reply:
Yeah, I think that's good.